src/Layers/xrRender/Blender_Recorder_R2.cpp

```diff
@@ -137,8 +137,10 @@ u32 CBlender_Compile::r_Sampler(
     {
 #if defined(USE_DX11)
         r_dx10Texture(_name, texture, true);
-#else
+#elif defined(USE_DX9) || defined(USE_OGL)
         i_Texture(dwStage, texture);
+#else
+#error No graphics API selected or enabled!
```

Comment:
Nested preprocessor directives should have an indent:
```suggestion
#   error No graphics API selected or enabled!
```
This applies to all `#error`s in this PR.